web/themes/custom/contribtracker/package.json

```diff
@@ -20,6 +22,14 @@
     "breakpoint-sass": "^3.0.0",
     "browser-sync": "^3.0.2",
     "cypress": "13.12.0",
+    "cypress-grep": "^2.0.1",
+    "cypress-xpath": "^2.0.1",
+    "mochawesome": "7.1.3",
+    "mochawesome-merge": "4.2.2",
+    "mochawesome-report-generator": "6.2.0",
+    "neat-csv": "5.1.0",
```

Comment:
This package has not seen an update in 3 years. I am not comfortable with introducing this as a dependency. There is https://www.npmjs.com/package/csv-parse instead but I think we can avoid using CSV entirely. How about storing the URL's as JSON instead?